l10n_it_account_stamp/models/account_move.py

```diff
@@ -8,7 +9,15 @@ class AccountMove(models.Model):
     _inherit = "account.move"
 
     tax_stamp = fields.Boolean(
-        "Tax Stamp", readonly=False, compute="_compute_tax_stamp", store=True
+        "Tax Stamp",
+        help="Tax stamp is applied to this invoice.",
+        readonly=False,
+        compute="_compute_tax_stamp",
+        store=True,
+    )
+    tax_stamp_line_present = fields.Boolean(
```

Comment:
https://github.com/OCA/l10n-italy/pull/4414#discussion_r1904602120

```suggestion
    tax_stamp__invoice_line_present = fields.Boolean(
```